packages/trip-form/src/TripOptions/OptionButton.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import * as S from "./styled";
+// eslint-disable-next-line prettier/prettier
+import type { ButtonProps } from "./Checkbox"
+
+export default function OptionButton({ onClick, label, children, className, selected, checked }: {
+  checked: boolean;
+  children?: React.ReactNode | string;
+  className?: string;
+  label: string;
+} & ButtonProps): React.ReactElement {
+  return (
+    <S.OptionButton className={className && className} onClick={onClick} selected={selected}>
```

Comment:
We don't need the `&&`.

Reply:
Removed in 775500dafb258ffc2fc5d431b8619ebe3f21b54f